core/BUILD

```diff
@@ -8,6 +8,26 @@ config_setting(
     constraint_values = ["@platforms//os:windows"],
 )
 
+# Define the instrument-hooks library separately to apply warning suppressions
+cc_library(
+    name = "instrument_hooks",
+    srcs = ["instrument-hooks/dist/core.c"],
+    hdrs = glob(["instrument-hooks/includes/*.h"]),
+    includes = ["instrument-hooks/includes"],
+    copts = select({
+        ":windows": [
+            "/wd4101",  # unreferenced local variable (equivalent to -Wno-unused-variable)
```

Comment:
open question: shouldn't this be the responsibility of an intrument-hooks' BUILD file ? It would make it easier the day we want to document the instrument hooks for homebrew usage